crates/re_viewer_context/src/selection_state.rs

```diff
@@ -268,14 +293,16 @@ impl ApplicationSelectionState {
     pub fn toggle_selection(&self, toggle_items: Selection) {
         re_tracing::profile_function!();
 
-        let mut toggle_items_set: HashMap<Item, Option<SelectedSpaceContext>> =
-            toggle_items.iter().cloned().collect();
+        let mut toggle_items_set: HashMap<Item, Option<SelectedSpaceContext>> = toggle_items
+            .iter()
+            .map(|(item, ctx)| (item.clone(), ctx.clone()))
+            .collect();
```

Comment:
kinda surprised `cloned` no longer worked? if both elements of the tuple are clonable then is the tuple